stable/kong/FAQs.md

```diff
@@ -0,0 +1,2 @@
+# Frequently Asked Questions (FAQs)
+
```

Comment:
Was additional content not yet committed here? It seems odd to add this with just the header.

Reply:
Yes, not ideal. Adding real soon but not in this PR. There is a humongous PR coming with loads of improvements.